src/main/java/com/puppycrawl/tools/checkstyle/site/PropertiesMacro.java

```diff
@@ -62,7 +63,10 @@ public class PropertiesMacro extends AbstractMacro {
     /** A newline with 20 spaces of indentation. */
     private static final String INDENT_LEVEL_20 = SiteUtil.getNewlineAndIndentSpaces(20);
 
-    /** The name of the current module being processed. */
+    /** The name of the token property being processed. */
+    private static final String TOKEN_PROPERTY = "tokens";
```

Comment:
TOKENS_PROPERTY

and fix same typo in javadoc comment.

Reply:
@romani Done